shoop/utils/models.py

```diff
@@ -11,6 +11,7 @@
 
 from django.db import models
 from django_countries.fields import Country
+from jinja2.filters import do_batch
```

Comment:
Sure. The reason it wasn't used in the first place was that, well, it didn't exist back then. (It was only added in 2576caba9614da5008b1b23754722b8c547da65f.)